demisto_sdk/commands/common/update_id_set.py

```diff
@@ -47,6 +47,14 @@
     PACKS_MAPPER_JSON_REGEX
 )
 
+CONTENT_ENTITIES = ['Integrations', 'Scripts', 'Playbooks', 'TestPlaybooks', 'Classifiers',
+                    'Dashboards', 'IncidentFields', 'IncidentTypes', 'IndicatorFields', 'IndicatorTypes',
+                    'Layouts', 'Reports', 'Widgets', 'Mappers']
+
+ID_SET_ENTITIES = ['integrations', 'scripts', 'playbooks', 'TestPlaybooks', 'Classifiers',
+                   'Dashboards', 'IncidentFields', 'IncidentTypes', 'IndicatorFields', 'IndicatorTypes',
+                   'Layouts', 'Reports', 'Widgets', 'Mappers']
```

Comment:
isn't that the same?

Reply:
No, you can see that the id_set names are different:
`'Integrations', 'Scripts', 'Playbooks'` and the content are `'Integrations', 'Scripts', 'Playbooks'`
Capital letters again lower case.

Didnt want to change the id_set entities since Im not sure it wont break BC